emcc.py

```diff
@@ -655,13 +655,13 @@ def is_minus_s_for_emcc(args, i):
     if debug_configure:
       tempout = '/tmp/emscripten_temp/out'
       if not os.path.exists(tempout):
-        open(tempout, 'w').write('//\n')
+        open(tempout, 'w', encoding='utf-8').write('//\n')
```

Comment:
this is a write, do we need to set the encoding here?

Reply:
(a write of a trivial string)

Reply:
oh; no it only actually needs to be set if there could be any unicode written.